src/vs/workbench/browser/parts/editor/editorPart.ts

```diff
@@ -979,11 +975,6 @@ export class EditorPart extends Part implements IEditorGroupsService, IEditorGro
 		this._onDidLayout.fire(dimension);
 	}
 
-	// {{SQL CARBON EDIT}} -- Allow editor titles to be refreshed to support tab coloring
```

Comment:
could you check if tab color is still working?

Reply:
The code was commented out so this doesn't functionally change anything. There are some bugs but they exist in the current master anyways.